test/Date/parseToStringResults.js

```diff
@@ -0,0 +1,49 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Date.parse must be able to parse the strings returned by Date.toString(), Date.toUTCString() and Date.toISOString()
+
+runTest("0001-10-13T05:16:33Z");
+runTest("0011-10-13T05:16:33Z");
+runTest("0111-10-13T05:16:33Z");
+runTest("1111-10-13T05:16:33Z");
+
+// test BC years
+let year1BC = new Date();
+let year11BC = new Date();

```

Comment:
Would be better to do `new Date("0001-10-13T05:16:33Z")` (with a specific initialization string) so that the values and output is more predictable.